### ISSUE TITLE
[BUG]Container version 2.4 is upgraded to community-2024-02-19 data is incompatible

### ISSUE THREAD

**Issue by Benxiaohai001** [cnosdb/cnosdb#1980]
### Describe the bug

Container version 2.4 is upgraded to community-2024-02-19 data is incompatible

### To Reproduce

```log
----------
The application panicked (crashed).
Message:  called `Result::unwrap()` on an `Err` value: SerdeMsgInvalid { err: "missing field `is_root_admin` at line 1 column 183" }
Location: /home/runner/work/cnosdb/cnosdb/meta/src/model/meta_admin.rs:118

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
                                ⋮ 9 frames hidden ⋮
  10: cnosdb::server::ServiceBuilder::create_meta::{{closure}}::h959a03ef5d5d6791
      at <unknown source file>:<unknown line>
  11: cnosdb::server::ServiceBuilder::build_query_storage::{{closure}}::hae8e8f3256a5972f
      at <unknown source file>:<unknown line>
  12: cnosdb::main::{{closure}}::h9f00d4c5371edbb3
      at <unknown source file>:<unknown line>
  13: cnosdb::main::hdf20e29e118ea588
      at <unknown source file>:<unknown line>
  14: std::sys_common::backtrace::__rust_begin_short_backtrace::h02fb8bd2c97b6475
      at <unknown source file>:<unknown line>
  15: std::rt::lang_start::{{closure}}::hd5e344bd86a80ffe
      at <unknown source file>:<unknown line>
  16: core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once::h8a913d44840f99aa
      at /rustc/b6a8c762eed0ae0383658c38d65cb91bbd9800a1/library/core/src/ops/function.rs:284
  17: std::panicking::try::do_call::h8ba215257741aa20
      at /rustc/b6a8c762eed0ae0383658c38d65cb91bbd9800a1/library/std/src/panicking.rs:552
  18: std::panicking::try::h85acc06cd9305921
      at /rustc/b6a8c762eed0ae0383658c38d65cb91bbd9800a1/library/std/src/panicking.rs:516
  19: std::panic::catch_unwind::he62c5d6e03102a53
      at /rustc/b6a8c762eed0ae0383658c38d65cb91bbd9800a1/library/std/src/panic.rs:142
  20: std::rt::lang_start_internal::{{closure}}::h2667b7e63fdb172f
      at /rustc/b6a8c762eed0ae0383658c38d65cb91bbd9800a1/library/std/src/rt.rs:148
  21: std::panicking::try::do_call::h9f7a182bfec92592
      at /rustc/b6a8c762eed0ae0383658c38d65cb91bbd9800a1/library/std/src/panicking.rs:552
  22: std::panicking::try::hbde58a5acc6a1792
      at /rustc/b6a8c762eed0ae0383658c38d65cb91bbd9800a1/library/std/src/panicking.rs:516
  23: std::panic::catch_unwind::ha7d493354abff086
      at /rustc/b6a8c762eed0ae0383658c38d65cb91bbd9800a1/library/std/src/panic.rs:142
  24: std::rt::lang_start_internal::h0a3ad413bf08e687
      at /rustc/b6a8c762eed0ae0383658c38d65cb91bbd9800a1/library/std/src/rt.rs:148
  25: main<unknown>
      at <unknown source file>:<unknown line>
  26: __libc_start_main<unknown>
      at <unknown source file>:<unknown line>
  27: _start<unknown>
      at /opt/zig-linux-x86_64-0.11.0/lib/libc/glibc/sysdeps/x86_64/start-2.33.S:120

Run with COLORBT_SHOW_HIDDEN=1 environment variable to disable frame filtering.
Run with RUST_BACKTRACE=full to include source snippets.
```

**Comment by roseboy-liu** [comment]
Before the stable version is released, version 2.4 does not guarantee upgrade compatibility.

**Comment by Benxiaohai001** [comment]
As explained above
